### DISCUSSION TITLE
Field name 'idx' is missing

### DISCUSSION THREAD

**Discussion by bombxdev** [nocodb/nocodb#1857] [General]
My table looks like this 
| idx | name | nick_name |
|--|--|--|
|  |  |  |

I found the problem when I look up on this table on nocodb ui that the field name 'idx' is missing.
I guess it  conflict with the field name of nocodb, isn't it ?

**Comment by o1lab** [comment]
Nope, idx might be you primary key - these are considered as `system fields`. You can enable to see them if you open `fields` menu option and enable system fields checkbox

**Reply by bombxdev** [reply]
Yeah, I found it. Thank you bro.
